guppylang/prelude/quantum.py

```diff
@@ -21,89 +21,110 @@ def quantum_op(op_name: str) -> ops.OpType:
     tys.Opaque(extension="prelude", id="qubit", args=[], bound=TypeBound.Any),
     linear=True,
 )
-class Qubit:
+class qubit:
     pass
 
 
 @guppy.hugr_op(quantum, quantum_op("H"))
-def h(q: Qubit) -> Qubit: ...
+def h(q: qubit) -> qubit:
```

Comment:
Weird, the pre-commit hook made me change this with `ruff format`... Version mismatch? Does the ci have a specific config that allows this?

Reply:
The pre-commit config needs to be updated, it's stuck at ruff v0.1.13. Try `pre-commit autoupdate`.

Reply:
Though it seems the github action is using a slightly older version too https://github.com/ChartBoost/ruff-action/blob/main/.pre-commit-config.yaml#L17

Perhaps it's best to run the pre-commit config on the CI instead of relying on a github action that will be out of our control.

Reply:
"on our end" meaning in local dev environments?

Reply:
Yes, at https://github.com/CQCL/guppylang/blob/main/.pre-commit-config.yaml#L28